libraries/standard/http/src/http_client.c

```diff
@@ -966,11 +1127,26 @@ HTTPStatus_t HTTPClient_Send( const HTTPTransportInterface_t * pTransport,
         LogError( ( "Parameter check failed: pRequestHeaders->pBuffer is NULL." ) );
         returnStatus = HTTP_INVALID_PARAMETER;
     }
+    else if( pRequestHeaders->headersLen < HTTP_MINIMUM_REQUEST_LINE_LENGTH )
+    {
+        LogError( ( "Parameter check failed: pRequestHeaders->headersLen "
+                    "does not meet minimum the required length. "
+                    "MinimumRequiredLength=%u, LastHeadersLength =%lu",
```

Comment:
Why use of _Last_ prefix for the length data?

Reply:
Not sure oversight thanks